player/lua/console.lua

```diff
@@ -446,16 +445,18 @@ local function populate_log_with_matches()
         local style = ''
         local terminal_style = ''
 
+        if i == selected_match or matches[i].index == default_item then
+            local color, alpha = mpv_color_to_ass(mp.get_property('osd-selected-color'))
+            local outline_color, outline_alpha =
+                mpv_color_to_ass(mp.get_property('osd-selected-outline-color'))
+            style = '{\\1c&H' .. color .. '&\\1a&H' .. alpha ..
+                    '&\\3c&H' .. outline_color .. '&\\3a&H' .. outline_alpha .. '&}'
+        end
         if matches[i].index == default_item then
-            style = styles.default_item
             terminal_style = terminal_styles.default_item
         end
         if i == selected_match then
-            local color, alpha = mpv_color_to_ass(mp.get_property('osd-selected-color'))
-            local outline_color, outline_alpha =
-                mpv_color_to_ass(mp.get_property('osd-selected-outline-color'))
-            style = style .. "{\\b1\\1c&H" .. color .. "&\\1a&H" .. alpha ..
-                             "&\\3c&H" .. outline_color .. "&\\3a&H" .. outline_alpha .. "&}"
+            style = style .. '{\\b1}'
```

Comment:
```suggestion
            style = style .. '{\\b}'
```
to reset?

Reply:
The selected item should be bold not reset.